### DISCUSSION TITLE
Painting backgrounds & the limits of clip dimensions

### DISCUSSION THREAD

**Discussion by eventualbuddha** [keiwando/toonsquid#642] [Q&A]
I’ve been trying to paint backgrounds that are wider than 4096 pixels (using pixel brushes) and running into the 4096x4096 limit. I’ve seen some mention of importing PNGs as a workaround. That would probably work, but it sounds kinda cumbersome. I’m open to it, but it’d be helpful if there were a guide for dealing with this sort of (probably common) problem.

On the issue of pixel limits, I get that there may need to be _some_ limit for the size due to resource constraints, but I don’t think it needs to be per dimension. Why not a limit on the total number of pixels? 4096x4096 = 16,777,216 pixels, so it feels like that should be the limit. I want to create e.g. a 4320x1080 background, which has 4,665,600 pixels. That’d be well under the hypothetical limit of 16.7M. It may be that “4096 pixels in each dimension” is an easier limit to understand, but “16.7M total pixels” isn’t _that_ much more difficult. Something to consider.

**Comment by SharpsterDraws** [comment]
If you work in vectors there is no limit to your size for the bkgd or turn you bkgd into a symbol and you can scale your art to any size…might not be ideal but is a good workaround.
